app/config/FrontendAppConfig.scala

```diff
@@ -91,6 +93,15 @@ class FrontendAppConfig @Inject()(runModeConfiguration: Configuration, environme
       .underlying.getString("urls.updateSchemeDetails")
   }"
 
+  def schemeDashboardUrl(request: DataRequest[_]): String = schemeDashboardUrl(request.psaId, request.pspId)
+
+  def schemeDashboardUrl(psaId: Option[PsaId], pspId: Option[PspId]): String =
+    (psaId, pspId) match {
+      case (Some(_), None) => managePensionsSchemeSummaryUrl
```

Comment:
I don't think this will work if user has both enrolments. For this you need to check the session data cache to see if user has chosen role. The code you need is in PODS-5414 - request.administratorOrPractitioner - once merged you can just use this.

Reply:
In this service, PSAs & PSPs have separate endpoints for all pages. So we don't really have to deal with a scenario where we have to display a page based on the user's selection in this service

Reply:
ok